setup.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+import hashlib
+import http
+import io
+import os.path
+import stat
+import sys
+import tarfile
+import urllib.request
+import zipfile
+from distutils.command.build import build as orig_build
+from distutils.core import Command
+from typing import Tuple
+
+from setuptools import setup
+from setuptools.command.install import install as orig_install
+
+TYPOS_VERSION = '1.0.9'
+POSTFIX_SHA256 = {
+    'linux': (
+        'x86_64-unknown-linux-gnu.tar.gz',
+        '',  # TODO: sha256 hexhash when we can generate it with release
+    ),
+    'darwin': (
+        'x86_64-apple-darwin.tar.gz',
+        '',  # TODO: sha256 hexhash when we can generate it with release
+    ),
+    'win32': (
```

Comment:
... `sys.platform()` uses `win32` for both 32-bit and 64-bit :/

(this confused me at first and looked it up)